src/Microsoft.Extensions.FileProviders.Physical/PhysicalFileInfo.cs

```diff
@@ -29,13 +29,12 @@ public PhysicalFileInfo(FileInfo info)
 
         public Stream CreateReadStream()
         {
-            // Note: Buffer size must be greater than zero, even if the file size is zero.
```

Comment:
Was this comment invalid? Because now `Math.Min((int)Length, 1024 * 64)` can return `0` which would result in an exception. Or can't `Length` be `0`? https://github.com/dotnet/corefx/blob/master/src/System.IO.FileSystem/src/System/IO/FileStream.cs#L85